.env

```diff
@@ -0,0 +1,10 @@
+MYSQL_DATABASE=bookstore_app
```

Comment:
Instead of adding .env file to git, create .env.template and provide only names of variables without values like
MYSQL_ROOT_PASSWORD=
MYSQL_LOCAL_PORT=

use git rm --cached .env to remove env file from git